src/utils/rootkeypackage_utils/src/rootkeypackage_parse.c

```diff
@@ -99,6 +99,43 @@ void ADUC_RootKeyPackage_Signature_DeInit(ADUC_RootKeyPackage_Signature* node)
     }
 }
 
+/**
+ * @brief Parses the isTest protected property in accordance with rootkeypackage.schema.json
+ *
+ * @param protectedPropertiesObj The root JSON object.
+ * @param[out] outPackage The root key package object to write parsed version data.
+ *
+ * @return The result.
+ */
+ADUC_Result RootKeyPackage_ParseIsTest(JSON_Object* protectedPropertiesObj, ADUC_RootKeyPackage* outPackage)
+{
+    ADUC_Result result = { .ResultCode = ADUC_GeneralResult_Failure, .ExtendedResultCode = 0 };
```

Comment:
nit: Should have the same error for test device being used in PROD same as line 104 in rootkey_workflow.c 

There's no reason this should ever be called unless we're in a testing environment (unless the function calling this one can never be called during normal workflow of the agent).